src/main/java/pt/lighthouselabs/obd/commands/ObdCommand.java

```diff
@@ -28,6 +28,7 @@
   protected String cmd = null;
   protected boolean useImperialUnits = false;
   protected String rawData = null;
+  protected long responseTimeDelay = 200;//automatic sampling rate detection coming
```

Comment:
Then make it `private final` and set it in a different constructor. In default constructor, use `200`.